otherlibs/unix/read_win32.c

```diff
@@ -103,3 +103,8 @@ CAMLprim value caml_unix_read_bigarray(value fd, value vbuf,
   }
   CAMLreturn(Val_int(numread));
 }
+
+CAMLprim value caml_unix_io_buffer_size(value unit) 
+{
+  return Val_long(UNIX_BUFFER_SIZE);
+}
```

Comment:
Since the constant is the same for both the Unix and Win32 backends, I would expect the definition of the function to be shared. However, I don't see right away a file where it could be put.